integreat_cms/cms/utils/matomo_api_manager.py

```diff
@@ -336,6 +344,14 @@ def get_visits_per_language(self, start_date, end_date, period):
                     # In this case, it matches the languages to their respective dataset (because the datasets are ordered)
                     for language, dataset in zip(languages, datasets)
                 ]
+                # The dataset for offline downloads
+                + [
+                    {
+                        "label": _("Offline Downloads"),
+                        "borderColor": colors.DEFAULT,
```

Comment:
```suggestion
                        "borderColor": next(color_cycle),
```
I would separate the offline download from the sum of all languages by using another color, so either choosing a random one from the dict or pre-defining a new constant for the offline downloads...